app/src/main/java/org/fossasia/openevent/activities/MainActivity.java

```diff
@@ -327,7 +360,8 @@ public void noInternet(NoInternetEvent event) {
     @Subscribe
     public void cantDownload(CantDownloadEvent event) {
```

Comment:
cantDownload was already there, didn't touch this. I'll remove it, it isn't needed.